src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/BodyModelBinder.cs

```diff
@@ -99,6 +96,8 @@ public BodyModelBinder(IList<IInputFormatter> formatters, IHttpRequestStreamRead
                 throw new ArgumentNullException(nameof(bindingContext));
             }
 
+            logger.TryingToBindModel(bindingContext);
```

Comment:
Where's Done logged?

Reply:
Since we don't know the outcome of these model binding attempts, I question the value of this logging. Our code fairly consistently brackets operations with "starting" and 'done".